### PR TITLE
docs: Update README and add GitHub action

### PR DESCRIPTION
**Pull request by xakraz**

### CODE REVIEW

**Comment by vincent-zurczak** [comment]
Hello @xakraz

Thank you for this contribution. :)
BTW, I did not know about the Github TOC action.

I start a review.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (vincent-zurczak) [inline README.md:1]: Why these two empty lines at the beginning?

File: README.md [hunk 2]
```diff
@@ -278,11 +286,16 @@ verify "there are <number> <resource-type> named '<regular-expression>'"
...
-This simple assertion may fail sometimes.  
-As an example, if you count the number of PODs, run your test and then kill the POD, they will still
-be listed, with the TERMINATING state. So, most of the time, you will want to verify the number of instances
+
+> :warning: This simple assertion may fail sometimes.
+>
+> As an example, if you count the number of PODs, run your test and then kill the POD, they will still
+be listed, with the `TERMINATING` state.
+>
+> So, most of the time, you will want to verify the number of instances
```
Comment (vincent-zurczak) [inline README.md:295]: To end with a dot.

File: README.md [hunk 3]
```diff
@@ -278,11 +286,16 @@ verify "there are <number> <resource-type> named '<regular-expression>'"
...
 with a given property value. Example: count the number of PODs with a given name pattern and having the `started` status.
-Hence this additional syntax.
+
+Hence this additional syntax (Using [next section](#verifying-property-values) documentation to verify additionnal properties):
```
Comment (vincent-zurczak) [inline README.md:298]: using (all in lower case)

**Review by vincent-zurczak — CHANGES_REQUESTED** [review]
Minor typos to fix.

File: README.md (hunk 1 above)
Comment (xakraz) [reply README.md:1]: Just an oversight 😅 

By default, `doctoc` puts the TOC at the beginning of the file.
I moved it down to the location of the previous toc and did not notice these 2 empty lines.

Fixing

File: README.md (hunk 2 above)
Comment (xakraz) [reply README.md:295]: ❓ The sentence continues the line after.
It seems to be a carriage return issue.

Fixing

**Review by vincent-zurczak — APPROVED** [review]

**Comment by vincent-zurczak** [comment]
Thanks. I have approved the changes. :+1: 
One last thing that is necessary so that I can merge is about [signing your commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits). I know it is a little bit annoying, but this is required for all the DETIK projects.

**Comment by xakraz** [comment]
@vincent-zurczak I setup my GPG key on GitHub and signed my commit (squashed by the way). 
I hope it is ok now 😄

**Comment by vincent-zurczak** [comment]
Thanks! :)

**Pull request merged**